README.md

```diff
@@ -6,13 +6,100 @@ via a dracut module.
 
-## Testing out the installer script
+## Using the installer on Fedora or RHEL CoreOS
+
+This installer is incorporated into Fedora CoreOS and RHEL CoreOS.
+There are ISO images that can be downloaded that will allow for an
+install to be performed on bare metal hardware either via ISO install
+or via a PXE install.
+
+#### Grab an ISO image and bare metal image
+
+For Fedora CoreOS you can currently grab an ISO image from the output
+of the current development pipeline located
+[here](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/).
+
+The ISO image can install in either legacy boot (BIOS) mode or in UEFI
+mode. You'll have to make sure you download the appropriate related
+image for the mode you'd like to use. It is a good idea to download
+them before doing the install as the artifacts server that is being
+used currently has very slow download speeds.
+
+For example download:
+
+- [fedora-coreos-29.731.iso](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731.iso)
+
+and one of the following two:
+
+- [fedora-coreos-29.731-metal-bios.raw.gz](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731-metal-bios.raw.gz)
+- [fedora-coreos-29.731-metal-uefi.raw.gz](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731-metal-uefi.raw.gz)
+
+**NOTE** Those links will quickly become broken because we prune
+         builds. You'll have to find your own links from the 
+         [latest directory](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/).
+
+#### Test an ISO based install
+
+You can test an install on a bare metal machine by burning the ISO to
+disk and booting it or using ISO redirection via a LOM interface.
+Alternatively you can use a VM like so:
+
+```
+virt-install --name cdrom --ram 4500 --vcpus 2 --disk size=20 --accelerate --cdrom /path/to/fedora-coreos-29.731.iso --network default
+```
+
+**NOTE** To test UEFI boot add `--boot uefi` to the CLI call
+
+One you have booted you will see a screen press `<TAB>` (isolinux) or
+`e` (grub) to edit the kernel command line. Add the parameters to the
+kernel command line telling it what you want it to do. For example:
+
+- `coreos.inst.install_dev=sda`
+- `coreos.inst.image_url=http://example.com/fedora-coreos-29.731-metal-bios.raw.gz`
+- `coreos.inst.ignition_url=http://example.com/config.ign`
+
+**NOTE** make sure to use a `metal-uefi` image if booting via UEFI
+
+Now press `<ENTER>` (isolinux) or `<CTRL-x>` (grub) to kick off the
+install. The install will occur on tty2 and there are very few good
+log statements or debug opportunities. Thee user experience here
+needs work and is tracked in https://github.com/coreos/coreos-installer/issues/5 .
+
+The install should progress and eventually reboot the machine. After
+reboot the machine will boot into the installed system and the
+embedded ignition config should run on first boot.
+
+#### Test a PXE based install
```

Comment:
Lets move this above ISO, since its an easier path lets encourage users to use it by putting it first.

Reply:
I can do that